src/status_im/chat/events/input.cljs

```diff
@@ -440,8 +441,11 @@
 
 (handlers/register-handler-fx
   :send-current-message
-  [(re-frame/inject-cofx :random-id)]
-  (fn [{{:keys [current-chat-id current-public-key] :as db} :db message-id :random-id current-time :now} _]
+  [(re-frame/inject-cofx :random-id)
+   (re-frame/inject-cofx :get-last-clock-value)
+   (re-frame/inject-cofx :get-stored-chat)]
```

Comment:
why do we need to take `:get-last-clock-value` and `:get-stored-chat`  from realm ? not from db?

Reply:
You are right that in this particular case (we are sending message from current chat), we actually don't need it.
The problem is that the same functionality (checking last message-id & getting chat from realm) is used in the receive process and there, it's possible that the chat (and so associated `last-clock-value`) is not in the app-db but it's in realm (as we don't hard-delete group chats, we just mark them as such in realm, this is important for leaving & re-joining public chats)